src/pages/test-report/components/examiner-takes-action/__tests__/eta.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { ComponentFixture, async, TestBed } from '@angular/core/testing';
+import { EtaComponent } from '../eta';
+import { IonicModule } from 'ionic-angular';
+import { testsReducer } from '../../../../../modules/tests/tests.reducer';
+import { StoreModule, Store } from '@ngrx/store';
+import { StoreModel } from '../../../../../shared/models/store.model';
+import { ToggleVerbalEta, TogglePhysicalEta } from '../../../../../modules/tests/test_data/test-data.actions';
+
+describe('LegalRequirementsComponent', () => {
+  let fixture: ComponentFixture<EtaComponent>;
+  let component: EtaComponent;
+  let store$: Store<StoreModel>;
+  let storeDispatchSpy: jasmine.Spy;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        EtaComponent,
+      ],
+      imports: [
+        IonicModule,
+        StoreModule.forRoot({ tests: testsReducer }),
+      ],
+    })
+      .compileComponents()
+      .then(() => {
+        fixture = TestBed.createComponent(EtaComponent);
+        component = fixture.componentInstance;
+        store$ = TestBed.get(Store);
+        storeDispatchSpy = spyOn(store$, 'dispatch');
```

Comment:
No need to assign the return value of of `spyOn`, you can call `expect(store$.dispatch).toHaveBeenCalled()`.